src/components/Calculator.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import './Calculator.css';
+
+const Calculator = () => {
+  const [displayValue, setDisplayValue] = useState('0');
+  const [previousValue, setPreviousValue] = useState(null);
+  const [operator, setOperator] = useState(null);
+
+  const handleDigitClick = (digit) => {
+    if (displayValue === '0') {
+      setDisplayValue(digit);
+    } else {
+      setDisplayValue((prevDisplay) => prevDisplay + digit);
+    }
+  };
+
+  const performOperation = () => {
+    const inputValue = parseFloat(displayValue);
+    const previous = previousValue;
+    let res = 0;
+    if (operator === '+') {
+      res = previous + inputValue;
+    } else if (operator === '-') {
+      res = previous - inputValue;
+    } else if (operator === '×') {
+      res = previous * inputValue;
+    } else if (operator === '÷') {
+      res = previous / inputValue;
+    } else if (operator === '+/-') {
+      res = -1 * previous;
+    } else if (operator === '%') {
+      res = previous / 100;
+    }
+
+    return res;
+  };
+
+  const handleOperatorClick = (nextOperator) => {
+    const inputValue = parseFloat(displayValue);
+
+    if (previousValue == null) {
+      setPreviousValue(inputValue);
```

Comment:
- [ ] [Optional][Highly recommended] Please remove all functions responsible for the calculator functionality because you will implement them in the following milestone using logic components provided by Microverse.